bin/workflows/pycbc_offline

```diff
@@ -0,0 +1,355 @@
+                                           extension='.html',
+                                           directory=rdir.base)
+kwds = { 'title' : 'Search Workflow Duration (Wall Clock Time)',
+        'caption' : "Wall clock start and end times for this invocation of "
+                    "the workflow. The command line button shows the "
+                    "arguments used to invoke the workflow creation script.",
+        'cmd' :' '.join(sys.argv), }
+save_fig_with_metadata(time_str, time_file.storage_path, **kwds)
+
+# Get segments and find where the data is
+wf.makedir(rdir['analysis_time/segment_data'])
+science_veto_name = 'segments-science-veto'
+primary_veto_name = 'segments-final-veto-group'
+secondary_vetoes_name = 'segments-veto-groups'
+nonscience_veto_names = [primary_veto_name, secondary_vetoes_name]
+
+science_seg_file, sci_segs, sci_seg_name = wf.get_science_segments(workflow,
+                                            rdir['analysis_time/segment_data'])
+
+runtime_names=[science_veto_name]
+in_workflow_names = nonscience_veto_names
+veto_cat_files = wf.get_files_for_vetoes(workflow,
+                                         rdir['analysis_time/segment_data'],
+                                         runtime_names=runtime_names,
+                                         in_workflow_names=in_workflow_names)
+
+sci_ok_seg_file, sci_ok_segs, sci_ok_seg_name = wf.get_analyzable_segments(\
+                                            workflow, sci_segs, veto_cat_files,
+                                            rdir['analysis_time/segment_data'])
+
+datafind_files, analyzable_file, analyzable_segs, analyzable_name = \
+                                           wf.setup_datafind_workflow(workflow,
+                                                     sci_ok_segs, "datafind",
+                                                     seg_file=science_seg_file)
+
+cum_veto_files, veto_names, ind_cats = wf.get_cumulative_veto_group_files(\
+                                            workflow, 'segments-veto-groups',
+                                            veto_cat_files,
+                                            rdir['analysis_time/segment_data'],
+                                            execute_now=False)
+
+final_veto_file, final_veto_name, ind_cats = \
+                                   wf.get_cumulative_veto_group_files(workflow,
+                                            'segments-final-veto-group',
+                                            veto_cat_files,
+                                            rdir['analysis_time/segment_data'],
+                                            execute_now=False)
+
+# setup gating files if provided
+gate_files = wf.setup_gating_workflow(workflow, output_dir="gating")
+
+# add the gate files to the jobs that use them
+for job in ['calculate_psd', 'inspiral', 'single_template',
+            'plot_singles_timefreq']:
```

Comment:
We need to delete this part and have it done automatically, but I'll merge that in later ...